infra/auth0/js/setCustomIdTokenClaims.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Handler that will be called during the execution of a PostLogin flow.
+ *
+ * @param {Event} event - Details about the user and the context in which they are logging in.
+ * @param {PostLoginAPI} api - Interface whose methods can be used to change the behavior of the login.
+ */
+exports.onExecutePostLogin = async (event, api) => {
+  api.idToken.setCustomClaim("gender", event.user.user_metadata.gender)
```

Comment:
`gender` får vi ikke lov til å sette av auth0, må f.eks. namespaces til `https://online.ntnu.no/gender` https://auth0.com/docs/secure/tokens/json-web-tokens/create-custom-claims#general-restrictions

Reply:
I think it's probably best to prefix all of the custom claims anyways. That's pretty common practice.

Suggest we do `https://online.ntnu.no/x` for all of them